dashboard/modules/job/sdk.py

```diff
@@ -64,6 +64,37 @@ def submit_job(
         runtime_env: Optional[Dict[str, Any]] = None,
         metadata: Optional[Dict[str, str]] = None,
     ) -> str:
+        """Submit and execute a job asynchronously.
+
+        When a job is submitted, it runs once to completion or failure. Retries or
+        different runs with different parameters should be handled by the
+        submitter. Jobs are bound to the lifetime of a Ray cluster, so if the
+        cluster goes down, all running jobs on that cluster will be terminated.
+
+        Example:
+            >>> job_submission_client.submit_job(
+            >>>     entrypoint="python script.py",
```

Comment:
is it possible to make these doctests self-contained? we have too many failing doctests already and ideally these are all executable. I realize you're referencing a script here, so that might be difficult, but at the very least we need imports and the definition of `job_submission_client`. New users won't know what that is and how to create it.